.github/workflows/ci.yml

```diff
@@ -355,4 +360,5 @@ jobs:
             nix run \
                 --accept-flake-config \
                 --extra-experimental-features '"nix-command flakes"' \
-                'github:flox/flox/${{ github.sha }}#packages.${{ matrix.system }}.flox-cli-tests'
+                'github:flox/flox/${{ github.sha }}#packages.${{ matrix.system }}.flox-cli-tests' \
+                -- -- -j 4
```

Comment:
nonblocking: could we set this higher? Or what's the justification for 4?

Reply:
4 seemed like a _safe_ number, given that free github runners have max 4 cores.
IIRC the tests are actually run on our hardware, so we can possibly set this higher as well -- 4 was mostly just a guess.

If we do see flaking in CI I'd rather reset this to run in sequence in CI and provide instruction to run them in parallel locally.